build/docker/Dockerfile

```diff
@@ -31,3 +32,22 @@ RUN cd /usr/src/gtest && cmake . && make && cp libgtest.a /usr/lib
 RUN git clone https://github.com/google/gtest-parallel.git
 
 ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+
+## Website build dependencies
+
+# Install Ruby-2.4.1
+RUN apt-get install -y
+RUN gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB && \
```

Comment:
I think this gpg-key might get expired after some time, so, should we avoid using key here?